homeassistant/components/philips_js/__init__.py

```diff
@@ -38,15 +38,22 @@
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Philips TV from a config entry."""
 
+    system = entry.data.get(CONF_SYSTEM)
```

Comment:
What's the type? Optional string?

Reply:
Do not all config entries have this item? Was it added at a later date than the first config entry of this  integration?

Reply:
Optional dict[str,Any]. Actual typed dict exist in lib. Not sure we need to type it 

I think it was missing in initial config flow.

Reply:
If we have a type for it it's nice to use that to explain the code for readers.

Reply:
Added